webob/response.py

```diff
@@ -1248,7 +1270,15 @@ def conditional_response_app(self, environ, start_response):
         * Range               (406 Partial Content; only on GET, HEAD)
         """
         req = BaseRequest(environ)
-        headerlist = self._abs_headerlist(environ)
+
+        # Build headerlist, if there is a Location header, make it absolute
+        headerlist = [
```

Comment:
note something about how this `_abs_headerlist` code is copied around / inlined from somewhere for future editors... possibly even leaving the original impl in place with tests, and a note about where it is inlined to... and the inline points should point back to the canonical impl